.travis.yml

```diff
@@ -0,0 +1,18 @@
+notifications:
+  slack: wtsi-cgpit:ptUMR1tkNyZJYd9TpGoss8WR
+  email: false
+
+sudo: false
+
+language: ruby
+
+services:
+  - docker
+
+script:
+ - echo 'Build and check docker image'
```

Comment:
Travis will only exit on error if the last step fails, it is recommended that you add the following as the first line:

```
 - set -e
```

As currently written the CI will pass even if the second to last command were to fail.